<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QWidget" name="Main">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>883</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>883</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Main</string>
  </property>
  <widget class="PlotWidget" name="graph_data" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="GLViewWidget" name="graph_3d" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>311</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="PlotWidget" name="graph_cost" native="true">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>311</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="graph_25d" native="true">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>311</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Iteraciones</string>
   </property>
   <widget class="QSpinBox" name="itr_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="value">
     <number>1500</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>80</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Alpha</string>
   </property>
   <widget class="QDoubleSpinBox" name="alpha_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QProgressBar" name="itr_progress">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>550</y>
     <width>551</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="itr_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Iteracion</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>150</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Datos</string>
   </property>
   <widget class="QLineEdit" name="file_path">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>data1.txt</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_buscar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>390</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Prueba</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>41</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Valor</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="test_input">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="result_output">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Resutado</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>87</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>270</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Theta</string>
   </property>
   <widget class="QDoubleSpinBox" name="theta1_input">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="theta2_input">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>540</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Iniciar</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GLViewWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph.opengl</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
